axelor-project/src/main/resources/views/Project.xml

```diff
@@ -381,6 +383,25 @@
         x-dirty="false" domain="self.project.id = :id" colSpan="6"/>
       <panel-dashlet action="action-project-view-sprint-allocation-lines"
         name="sprintAllocationLinePanel" colSpan="12"/>
+      <panel-dashlet action="action-project-method-tasks-per-sprint"
+        name="tasksPerSprintPanel" colSpan="12"/>
+    </panel>
+
+  </form>
+
+  <form model="com.axelor.utils.db.Wizard" title="Attach tasks to sprint"
+    name="attach-tasks-to-sprint-wizard-form">
+
+    <panel name="mainPanel">
+      <field name="sprint" title="Sprint" type="many-to-one"
+        target="com.axelor.apps.project.db.Sprint" domain="self.project.id = :_projectId"
+        colSpan="6"/>
+      <field name="projectTasks" title="Project tasks" type="many-to-many"
```

Comment:
Create a new grid view for ProjectTask with 
ticketNumber (if __config__.app.getApp('project')?.getIsEnablePerProjectTaskSequence()), 
name, 
priority (add the same hilite that in project-task-grid)
taskDate
sprint

And use it in this view please